src/App.tsx

```diff
@@ -10,6 +11,7 @@ const App = () => {
     <BrowserRouter>
       <Header />
       <Routes>
+        <Route path={ROUTES.HOME} element={<HomePage />} />
         <Route path={ROUTES.HOME} element={<div>Home</div>} />
```

Comment:
```suggestion
        <Route path={ROUTES.HOME} element={<HomePage />} />
```

기존 HOME은 그대로 있어요! 제거부탁드려용

Reply:
확인했습니다. 제거했어요:)